verifier/builtins.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * ZCrypto Copyright 2017 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package verifier
+
+import "github.com/zmap/zcrypto/x509"
+
+// Built-in verifiers representing common root stores.
+var (
+	// NSS is a Verifier mimicing the validation used in Firefox.
```

Comment:
https://img.memesuper.com/77a96c93ce547992cb7f93ffb2f9ac65_you-had-a-spelling-error-but-you-made-me-laugh-approving-spelling-error-meme_600-600.jpeg